src/libYARP_OS/include/yarp/os/Os.h

```diff
@@ -189,6 +189,20 @@ namespace yarp {
          * @since YARP 2.3.70
          */
         YARP_OS_API int fork(void);
+
+       /**
+        * @brief Toggle the OS energy saving feature
+        *
+        * This function toggle the state of the energy saving feature
+        * that may affect YARP calls
+        *
+        * For example, in macOS this function toggles the state of the AppNap
+        * feature, which put the process into an idle state.
+        *
+        * @param enable true if the energy saving mode should be enabled. False otherwise.
+        * @since YARP 2.3.72
+        */
+        YARP_OS_API void setEnergySavingModeEnabled(bool enabled);
```

Comment:
@drdanz already commented for the name of this method, but I put my two cents here as well.
I may be picky here, but I would not use `set` and `Enable` in an `ON/OFF` method. Simply using `setEnergySavingMode(bool status)` should be self-explanatory: pass `true/false` to enable/disable it.

Reply:
Then I would prefer `setEnergySavingModeState(bool status)`,
but please, decide together with @drdanz  😛

Reply:
For me `setEnergySavingModeState` is fine.
I understand that I can set a status of something passing a boolean `ON/OFF` parameter.
@drdanz is that ok for you?